packages/frameworks-astro/index.ts

```diff
@@ -1,3 +1,33 @@
+/**
+ * :::caution
+ * `@auth/astro` is currently experimental. Be aware of breaking changes between versions.
+ * :::
+ *
+ * Astro Auth is the official Astro integration for Auth.js.
+ * It provides a simple way to add authentication to your Astro site in a few lines of code.
+ *
+ * ## Installation
+ *
+ * :::info
+ * `@auth/astro` requires building your site in `server` mode with a platform adaper like `@astrojs/node`.
+ * ```js
+ * // astro.config.mjs
+ * export default defineConfig({
+ *   output: "server",
+ *   adapter: node({
+ *     mode: 'standalone'
+ *   })
+ * });
+ * ```
+ * :::
+ *
+ * ```bash npm2yarn2pnpm
+ * npm install @auth/core @auth/sveltekit
```

Comment:
You didn't mean to put SvelteAuth

Reply:
AHH, thanks for catching this :sweat_smile: